### ISSUE TITLE
Are you still developing this project

### ISSUE THREAD

**Issue by alvipan** [alaminsiddique/alpha_faucet_script#1]
Are you still developing this project? I want to help design the template.

**Comment by 0xArrow** [comment]
No, I am not actively developing it anymore since I got busy with other projects. You can use the code and make your own version with your template. :)
